dev/plugins/hu.elte.txtuml.diagnostics/pom.xml

```diff
@@ -11,4 +11,16 @@
 	</parent>
 	<artifactId>hu.elte.txtuml.diagnostics</artifactId>
 	<packaging>eclipse-plugin</packaging>
-</project>
```

Comment:
Please revert this in `hu.elte.txtuml.api.model.execution`, as `HttpServer` – the class which caused the problematic warnings –, is not needed there anymore.